web/templates/recipes/index.html

```diff
@@ -13,6 +14,11 @@ <h2>Search for Recipes</h2>
             Enter ingredients below, then press search to find recipes containing those ingredients!
         </p>
     </div>
+    <center> <!-- Error Alert -->
+        <div id="errorAlert" class="alert alert-danger center-block alert-dismissible fade show" role="alert">
+            <button id="errorAlertButton" type="button" class="btn-close alert" data-dismiss="alert" aria-label="Close"></button>
+        </div>
+    </center>
```

Comment:
There's probably a class you can add to center it rather than nesting it in a `center` element. Less nesting = more readable. I don't know which class that would be without playing around with it. Please don't spend a lot of time trying to do that. I'm just pointing it out for the sake of teaching.